src/test/regress/citus_tests/test/test_columnar.py

```diff
@@ -0,0 +1,113 @@
+import psycopg
+import pytest
+
+
+def test_freezing(coord):
+    coord.configure("vacuum_freeze_min_age = 50000")
+    coord.configure("vacuum_freeze_table_age = 50000")
```

Comment:
what about ?
```
synchronous_commit = off
fsync = off
```

Reply:
`fsync = false` is done globally already (See `initdb` method of `Postgres` class in `common.py`). Setting `syncronous_commit` in addition to that shouldn't help much, because we're already not waiting for disk flushes at all anyway.

Reply:
Is there an API to give multiple configurations needed for a Citus instance at once?
For example, in my TAP test I have these options:

max_connections = 100
max_wal_senders = 100
max_replication_slots = 100
citus.enable_change_data_capture = on
citus.override_table_visibility = off